{
  "contest": 129,
  "handle": "unforgiven",
  "address": "0x6e6c53f81e310d53387dc2166ee9a632cf768fa1",
  "risk": "3",
  "title": "startErUp() in RubiconRouter is callable by anyone which can cause DOS or griefing or fund loss if attacker calls it",
  "issueId": 230,
  "issueUrl": "https://github.com/code-423n4/2022-05-rubicon-findings/issues/230"
}